### PR TITLE
Initial implementation of circle job.

### LINKED ISSUE #4: Setup CI

### PR DESCRIPTION
**Pull request by jim**
This adds a sample CI configuration to the project.

Closes #4

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        environment:
+        - POSTGRES_DB: bork-local
+        - POSTGRES_PASSWORD: postgres
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - go-mod-sources-v1-{{ checksum "go.sum" }}
+      - run: wget -qO- https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/6.5.3/flyway-commandline-6.5.3-linux-x64.tar.gz | tar xvz && sudo ln -s `pwd`/flyway-6.5.3/flyway /usr/local/bin
```
Comment (eeeady) [inline .circleci/config.yml:47]: Is this a common utility we're using? It might be worth putting this in a docker container for usage.

**Review by eeeady — COMMENTED** [review]
I have an in-line question and... I was wondering how you consume the test results. We have some examples for how to get JUNIT reports out and into circleci.

File: .circleci/config.yml (hunk 1 above)
Comment (jim) [reply .circleci/config.yml:47]: [There is a docker image that we can probably leverage here](https://github.com/flyway/flyway-docker), although I haven't looked into how to do that in Circle yet. Do you think that is a better approach than installing it into this image?

**Comment by jim** [comment]
@eeeady I will add the Junit test result export to this.

**Pull request closed without merging**